android/app/src/main/java/org/fossasia/openevent/dbutils/RealmDatabaseMigration.java

```diff
@@ -2,10 +2,11 @@
 
 import io.realm.DynamicRealm;
 import io.realm.RealmMigration;
+import io.realm.RealmSchema;
 
 public class RealmDatabaseMigration implements RealmMigration {
 
-    public static final long DB_VERSION = 0;
+    public static long DB_VERSION = 0;
```

Comment:
The value 0 needs to be bumped at declaration only. It will not be reassigned, that's why final

0 is to be replaced by 1